pkg/services/govmomi/vcenter/clone_test.go

```diff
@@ -0,0 +1,153 @@
+/*
+Copyright 2019 The Kubernetes Authors.
```

Comment:
should be `Copyright 2020`

Reply:
Right, should be fixed now, thanks :+1:   PTAL @gab-satchi @randomvariable